packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1687,8 +1688,31 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 summaryLogger,
             });
 
+            const summary = summarizeResult.summary.tree;
+            const summaryTree = summary[channelsTreeName] as ISummaryTree;
+            let all = 0;
+            let unreferenced = 0;
```

Comment:
Rename these to the actual properties they will become, especially since the scope of this function is already really big.

```suggestion
            let dataStoreCount = 0;
            let unreferencedDataStoreCount = 0;
```